src/30_Advanced/Favorite_Button.html

```diff
@@ -0,0 +1,133 @@
+<!--
+  ## Introduction
+
+  This sample demonstrates how to implement a favorite/like/bookmark button in AMP. Our implementation features:
+  
+  * Shows the correct icon based on whether the user already liked an item or not. This works if the AMP is saved 
```

Comment:
if the AMP is saved  -> if the AMP is served

Reply:
done